src/BloomBrowserUI/bookEdit/toolbox/toolbox.ts

```diff
@@ -382,6 +382,11 @@ function showToolboxChanged(showing: boolean): void {
     if (currentTool) {
         if (showing) currentTool.hideTool();
         else currentTool.showTool();
+    } else {
```

Comment:
It should already show the most recently shown tool for the current book if there is one; this change is only for NEW books. Do we want those to remember which tool was active in the previously selected book? But anything but talking book may not even be turned on for a new book...